src/lib.rs

```diff
@@ -482,7 +673,11 @@ fn _torchft(m: &Bound<'_, PyModule>) -> PyResult<()> {
     m.add_class::<ManagerServer>()?;
     m.add_class::<ManagerClient>()?;
     m.add_class::<LighthouseServer>()?;
+    m.add_class::<LighthouseClient>()?;
     m.add_class::<QuorumResult>()?;
+    m.add_class::<PyQuorumMember>()?;
+    m.add_class::<PyQuorum>()?;
+    m.add_class::<PyTimestamp>()?;
```

Comment:
Don't we need to rename these fields as well? Are we overriding the Py prefix somewhere?

Would also be good to add a test with asserts on the return types of the objects in Python

Reply:
OK, figured out a way to disambiguate same naming.